dashboard/test/ui/README.md

```diff
@@ -46,7 +46,10 @@ You can find the values for these settings in your saucelabs account settings (`
 If you want to run tests on saucelabs against localhost you need to set up your tunnel:
 
 1. Login to Saucelabs and download the [tunnel](https://app.saucelabs.com/tunnels)
-2. Start the tunnel via `~/bin/sc/ -u $SAUCELABS_USERNAME -k SAUCELABS_ACCESS_KEY`. Notes: The link above has an example command line with your credentials that you can copy. `~/bin/sc/` should be replaced with the path that points to the tunnel software downloaded in step 1.
+2. Start the tunnel via `~/bin/sc/ -u $SAUCELABS_USERNAME -k SAUCELABS_ACCESS_KEY`
+   - Notes: The link above has an example command line with your credentials that you can copy.
+     - `bin/sc` should be replaced with the path that points to the tunnel software downloaded in step 1.
+     - The unique tunnel identifier `-i [tunnel_id]` included in that command can be safely ignored; if you leave it in, you'll also need to set the `tunnelIdentifier` option in the `sauce_capabilities` config. See [Using Sauce Connect Tunnel Identifiers](https://wiki.saucelabs.com/display/DOCS/Using+Sauce+Connect+Tunnel+Identifiers#UsingSauceConnectTunnelIdentifiers-TheBasicsofUsingTunnelIdentifiers) for more details.
```

Comment:
```suggestion
     - The unique tunnel identifier `-i $TUNNEL_ID` included in that command can be safely ignored; if you leave it in, you'll also need to set the `tunnelIdentifier` option in the `sauce_capabilities` config. See [Using Sauce Connect Tunnel Identifiers](https://wiki.saucelabs.com/display/DOCS/Using+Sauce+Connect+Tunnel+Identifiers#UsingSauceConnectTunnelIdentifiers-TheBasicsofUsingTunnelIdentifiers) for more details.
```

Reply:
this implies to me that folks have set TUNNEL_ID as a bash environment variable, which I don't think we want to implay

Reply:
yeah, that makes sense -- i really only care that we're consistent with the original command: `bin/sc -u $SAUCELABS_USERNAME -k SAUCELABS_ACCESS_KEY`

Reply:
so we could change the original command to `bin/sc -u [saucelabs_username] -k [saucelabs_access_key]` instead?